etcd/v2/kv_etcd.go

```diff
@@ -1,1003 +0,0 @@
-package etcdv2
```

Comment:
why was this deleted? just curious

Reply:
I asked a question in the channel, and @adityadani confirmed that it is ok to deprecate the etcd2 reference in our code.